app/services/exchange_rates/spot_exchange_rates_service.rb

```diff
@@ -0,0 +1,34 @@
+module ExchangeRates
+  class DataNotFoundError < StandardError; end
+
+  class SpotExchangeRatesService
+    def initialize(sample_date, download:)
+      @sample_date = sample_date
+      @download = download
+    end
+
+    def call
+      ExchangeRates::UpdateCurrencyRatesService.new(sample_date, sample_date, ExchangeRateCurrencyRate::SPOT_RATE_TYPE).call if download
```

Comment:
this is how it was recently refactored by Will so I don't want to change it because it would break monthly rates. Sample date is the same for spot rates because we want the rates the day the job was run and we want to save them with the same date

Reply:
What i mean is sample date doesnt mean anything to me? Is it a random date/ todays date i think this should be changed here